src/test/java/com/wikia/webdriver/testcases/adstests/TestAdsPremiumPrerollMOATTracking.java

```diff
@@ -0,0 +1,30 @@
+package com.wikia.webdriver.testcases.adstests;
+
+import com.wikia.webdriver.common.core.annotations.NetworkTrafficDump;
+import com.wikia.webdriver.common.dataprovider.ads.AdsDataProvider;
+import com.wikia.webdriver.common.templates.TemplateNoFirstLoad;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.adsbase.AdsOoyalaObject;
+import org.testng.annotations.Test;
+
+public class TestAdsPremiumPrerollMOATTracking extends TemplateNoFirstLoad {
+  private static final String TURN_ON_MOAT = "InstantGlobals.wgAdDriverMoatTrackingForFeaturedVideoAdCountries=[XX]";
+  private static final String IGNORE_SAMPLING = "ignored_samplers=moatTrackingForFeaturedVideo";
+  private static final String MOAT_VIDEO_TRACKING_URL = "z.moatads.com/wikiaimajsint377461931603/moatvideo.js";
+
+  @NetworkTrafficDump(useMITM = true)
+  @Test(
+      dataProviderClass = AdsDataProvider.class,
+      dataProvider = "adsPremiumPreroll",
+      groups = {
+        "AdsOoyalaPrerollOasis", "AdsOoyalaPrerollMercury",
+        "AdsPremiumPrerollMOATTrackingOasis", "AdsPremiumPrerollMOATTrackingMobile"
```

Comment:
Test will not start on Mercury please set emulator = Emulator.GOOGLE_NEXUS_5 for mobile tests